lib/compiler.js

```diff
@@ -204,9 +209,10 @@ function tryReadDir() {
 }
 
 function buildAllModelInstructions(rootDir, modelsConfig, sources,
-    modelDefinitions) {
-  var registry = verifyModelDefinitions(rootDir, modelDefinitions) ||
-    findModelDefinitions(rootDir, sources);
+    modelDefinitions, scriptExtensions) {
+  var registry = verifyModelDefinitions(rootDir, modelDefinitions,
+                  scriptExtensions) ||
+                  findModelDefinitions(rootDir, sources, scriptExtensions);
```

Comment:
The indentation makes it difficult to distinguish function parameters from the second function call. I am proposing the following style:

```js
  var registry = verifyModelDefinitions(rootDir, modelDefinitions,
                                        scriptExtensions);

  if (!registry) {
    registry = findModelDefinitions(rootDir, sources, scriptExtensions);
  }
```